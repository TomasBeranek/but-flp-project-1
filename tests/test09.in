A,B
a,b,c,d,e
A
A->aabb
A->cB
B->bdeB
B->#
