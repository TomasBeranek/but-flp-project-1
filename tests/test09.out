0,1,2,3,4,5,6,7
abcde
0
1,7
0,a,4
0,c,1
1,b,2
2,d,3
3,e,1
4,a,5
5,b,6
6,b,7
